config/derive/src/lib.rs

```diff
@@ -199,11 +199,16 @@ fn impl_get_doc_recursive(
         .map(|(((ident, inner_thing), documentation), ty)| {
             if inner_thing {
                 quote! {
-                    [stringify!(#ident)] => Some(#documentation),
+                    [stringify!(#ident)] => {
+                        let curr_doc = #documentation;
+                        let inner_docs = <#ty as iroha_config::Configurable>::get_inner_docs();
+                        let total_docs = format!("{}\n\nHas following fields:\n\n{}\n", curr_doc, inner_docs);
```

Comment:
Sorry, I just remembered. What if you try to use [`concat`](https://doc.rust-lang.org/std/macro.concat.html)? Then you can leave `&'static str`, I think

Reply:
I don't mind either way. The only place affected is kagami and it had a `to_owned()` somewhere, so semantically what we want is closer to a `String`. 

At some point we should consider making them immutable strings that don't store the capacity, but that's a bit far ahead.